google/resource_compute_region_instance_group_manager.go

```diff
@@ -145,6 +149,18 @@ func resourceComputeRegionInstanceGroupManager() *schema.Resource {
 					},
 				},
 			},
+
+			"distribution_policy": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
```

Comment:
I don't think this needs to be computed, does it?

Reply:
I'd have to go back and test with/without it, but I believe it needs to be computed because while the API will let you pass in, say, `us-central1-a`, whenever you retrieve the info for the rIGM, you _will_ get back the full zone URL i.e. `https://googlecloud.com/ur_project/zones/us-central1-a`, and then TF would freak out thinking there needed to be an update.

Let me try to retest that one.  I'm also open to ideas on generating the full one URL behind-the-scenes, but it seemed like a bunch of extra work at the time, and I wasn't really familiar with a way to easily nab the current project, etc, to build it.

Reply:
Nah, that's what the diffsuppress is for. Computed means that if it isn't set, the API will provide a value (so don't show a diff when the config doesn't have a value for that field).

Reply:
So, I went back.  This is actually required.

If we use the beta API version, the API actually fills in the distribution policy for you, based on -- what appears to be -- the zones the rIGM seeds itself with during creation.